kernel/solo5.h

```diff
@@ -136,8 +165,13 @@ uint64_t solo5_clock_monotonic(void);
 uint64_t solo5_clock_wall(void);
```

Comment:
That is good to hear - related to above, maybe it makes sense to explicitly document in `solo5.h` that fatal errors may occur in each call, which lead to an assertion failure (or `exit`)!?